pytest_terraform/tf.py

```diff
@@ -386,11 +391,15 @@ def create(self, request, module_dir):
         if self.teardown_config != td.OFF:
             request.addfinalizer(self.tear_down)
         try:
-            test_api = self.runner.apply()
-            tfstatejson = test_api.save()
-            self.config.hook.pytest_terraform_modify_state(tfstate=tfstatejson)
-            test_api.update(tfstatejson)
-            test_api.save(module_dir.join("tf_resources.json"))
+            state = self.runner.apply()
+            statejson = state.export()
```

Comment:
state_json

Reply:
made the changes but never pushed :sweat_smile: